README.md

```diff
@@ -245,6 +245,22 @@ including conditionals, loops, and functions. Boilerplate also includes several
    return str2;
    // boilerplate-snippet: foo
    ```
+1. `downcase STRING`: Convert `STRING` to lower case. E.g. "FOO" becomes "foo".
+1. `upcase STRING`: Convert `STRING` to upper case. E.g. "foo" becomes "FOO".
+1. `capitalize STRING`: Capitalize the first letter of each word in `STRING`. E.g. "foo bar baz" becomes "Foo Bar Baz".
+1. `replace OLD NEW`: Replace the first occurrence of `OLD` with `NEW`. This is a literal replace, not regex.
+1. `replaceAll OLD NEW`: Replace all occurrences of `OLD` with `NEW`. This is a literal replace, not regex.
+1. `trim STRING`: Remove leading and trailing whitespace from `STRING`.
+1. `round FLOAT`: Round `FLOAT` to the nearest integer. E.g. 1.5 becomes 2.
+1. `ceil FLOAT`: Round up `FLOAT` to the nearest integer. E.g. 1.5 becomes 2.
+1. `floor FLOAT`: Round down `FLOAT` to the nearest integer. E.g. 1.5 becomes 1.
+1. `dasherize STRING`: Convert `STRING` to a lower case string separated by dashes. E.g. "foo Bar baz" becomes
+   "foo-bar-baz".
+1. `snakeCase STRING`: Convert `STRING` to a lower case string separated by underscores. E.g. "foo Bar baz" becomes
+   "foo_bar_baz".
+1. `camelCase STRING`: Convert `STRING` to a camel case string. E.g. "foo Bar baz" becomes "FooBarBaz".
+1. `camelCaseLower STRING`: Convert `STRING` to a camel case string where the first letter is lower case. E.g.
+   "foo Bar baz" becomes "fooBarBaz".
```

Comment:
Most are based on functions built into Ruby/Rails/Jekyll and my experience using [giter8](https://github.com/foundweekends/giter8).